Assets/LeapMotion/Core/Scripts/XR/LeapXRCameraControl.cs

```diff
@@ -0,0 +1,89 @@
+/******************************************************************************
+ * Copyright (C) Leap Motion, Inc. 2011-2017.                                 *
+ * Leap Motion proprietary and  confidential.                                 *
+ *                                                                            *
+ * Use subject to the terms of the Leap Motion SDK Agreement available at     *
+ * https://developer.leapmotion.com/sdk_agreement, or another agreement       *
+ * between Leap Motion and you, your company or other organization.           *
+ ******************************************************************************/
+
+using UnityEngine;
+using UnityEngine.Rendering;
+using System;
+
+namespace Leap.Unity {
+
+  /// <summary>
+  /// Provides a handful of VR related camera utilities, such as controlling IPD and
+  /// camera distance.
+  /// </summary>
+  public class LeapXRCameraControl : MonoBehaviour {
+
+    private Matrix4x4 _finalCenterMatrix;
+
+    private LeapDeviceInfo _deviceInfo;
+
+    void Start() {
+      _deviceInfo = LeapDeviceInfo.GetLeapDeviceInfo();
+    }
+
+    private Camera _cachedCamera;
+    private Camera _camera {
+      get {
+        if (_cachedCamera == null) {
+          _cachedCamera = GetComponent<Camera>();
+        }
+        return _cachedCamera;
+      }
+    }
+
+    [SerializeField]
+    private EyeType _eyeType = new EyeType(EyeType.OrderType.CENTER);
+
+    /// <summary>
+    /// Called during the left eye camera's OnPreRender Unity callback.
+    /// </summary>
+    public static Action<Camera> OnLeftPreRender;
```

Comment:
We can probably remove these since they are not used anywhere and the reason they were created is no longer needed.

Reply:
good call